demo/src/routes/menu/CollapsibleSection.svelte

```diff
@@ -0,0 +1,50 @@
+<script>
+	import {collapseVerticalTransition} from '@agnos-ui/core/services/transitions/bootstrap';
+	import {createTransition} from '@agnos-ui/core/services/transitions/baseTransitions';
+	import {writable} from '@amadeus-it-group/tansu';
+
+	export let headerText;
+	const paramTransition$ = writable(collapseVerticalTransition);
+	const paramAnimation$ = writable(true);
+	const paramVisible$ = writable(false);
+
+	const {
+		stores: {visible$},
+		api: {toggle},
+		directives: {directive},
+	} = createTransition({
+		props: {
+			animation: paramAnimation$,
+			visible: paramVisible$,
+		},
+	});
+</script>
+
+<div class="mb-2">
+	<button class="btn btn-light btn-lg p-1 d-flex align-items-center btn-header" aria-expanded={$visible$} on:click={() => toggle()}
```

Comment:
I would probably add `justify-content-between` and `w-100` here to give more order in the layout

Reply:
i like it, good suggestion!